blocks/plans-quote/costco-promo.js

```diff
@@ -1,27 +1,26 @@
 import {
   PET_PLANS_ANNUAL_URL,
-  LS_KEY_FIGO_COSTCO,
+  LS_KEY_COSTCO_FIGO,
 } from '../../scripts/24petwatch-utils.js';
 import APIClient, { getAPIBaseUrl } from '../../scripts/24petwatch-api.js';
 import { getConfigValue } from '../../scripts/configs.js';
 
 export const COSTCO_FIGO_PROMO_ITEMS = {
   policyIdKey: 'poid',
-  subId: 'SBSUN000016',
+  subId: 'COSTCOFIGO',
```

Comment:
why not create const for this one as well to import in

Reply:
The object is a const and is exported.